api/v1/_submissions/BackendSubmissionsHandler.inc.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file api/v1/_submissions/BackendSubmissionsHandler.inc.php
```

Comment:
Excellent, it looks like you've rebased that branch on mine. Is that right? If so, that'd be great.